bundles/AdminBundle/Resources/public/js/pimcore/asset/embedded_meta.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Enterprise License (PEL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PEL
+ */
+
+pimcore.registerNS("pimcore.asset.embedded_meta");
+pimcore.asset.embedded_meta = Class.create({
```

Comment:
Please call that `pimcore.asset.embedded_meta_data` and also rename the file - we should be consistent with the naming structure.